scripts/deploy.sh

```diff
@@ -14,7 +15,7 @@ OSAD_DIR='/opt/rpc-extras/os-ansible-deployment'
 RPCD_DIR='/opt/rpc-extras/rpcd'
 
 # begin the bootstrap process
-cd "${OSAD_DIR}"
+cd ${OSAD_DIR}
```

Comment:
In this case it doesn't really change anything - although the change itself seems unnecessary.